apps/condo/domains/banking/tasks/importBankTransactionsFrom1CClientBankExchange.js

```diff
@@ -1,12 +1,13 @@
 const fetch = require('isomorphic-fetch')
 const { get } = require('lodash')
+const isEmpty = require('lodash/isEmpty')
```

Comment:
It's already import from `lodash`. You can just append isEmpty to the previous line
```suggestion
const { get, isEmpty } = require('lodash')
```